include/zigbee/zigbee_app_utils.rst

```diff
@@ -135,7 +135,12 @@ Complete zigbee_default_signal_handler implementation
 
 In its complete implementation, the `zboss_signal_handler()`_ allows the application to control a broader set of basic functionalities, including joining, commissioning, and network formation.
 
-The following sections describe the logic implemented inside :c:func:`zigbee_default_signal_handler`.
+.. figure:: /images/zigbee_signal_handler_overview.svg
```

Comment:
I do not understand this figure: so there are four options after the BDB initialization (new device scenario, commissioned device scenario, network joining, network formation) ?
Is not it that way, that the scenarios are enclosed inside the same block as the BDB initialization?

Reply:
I'm not sure if I am a fan of arrows partially hidden behind boxes